src/views/LimitedRestorationToFull.vue

```diff
@@ -42,21 +46,36 @@
             <h1>Review and Certify</h1>
           </header>
 
-          <RestorationSummary
-            class="mt-10"
-            :validate="getAppValidate"
-          />
+          <div class="document-info py-4">
+            Review and certify the information in your application. If you need to change or complete anything,
+            return to the previous step to make the necessary change.
+          </div>
+
+          <!-- Applicant list -->
+          <v-card id="people-and-roles-vcard" flat class="mt-6">
+            <!-- Header -->
+            <div class="section-container header-container">
+              <v-icon color="appDkBlue">mdi-account-multiple-plus</v-icon>
+              <label class="font-weight-bold pl-2">{{ orgPersonLabel }} Information</label>
+            </div>
+            <div no-gutters class="mt-4 section-container">
+              <ListPeopleAndRoles
+                :isSummaryView="true"
+                :showDeliveryAddressColumn="false"
+                :showRolesColumn="false"
+                :showEmailColumn="true"
+              />
+            </div>
+          </v-card>
 
           <YourCompanySummary class="mt-10" />
 
-          <CurrentDirectors class="mt-10" />
-
-          <ListPeopleAndRoles class="mt-10" />
-
           <DocumentsDelivery
             class="mt-10"
             sectionNumber="1."
             :validate="getAppValidate"
+            :userEmailOptional="userEmailOptional"
+            :applicantEmail="applicantEmail"
             @valid="setDocumentOptionalEmailValidity($event)"
           />
```

Comment:
Here is the screenshot of Sumamry page:

![image](https://user-images.githubusercontent.com/116035339/229647903-5b1d5357-34f8-4c6d-9e27-1862fbeb761d.png)